mkdocs-base.yml

```diff
@@ -85,7 +85,7 @@ plugins:
   with-pdf: # https://github.com/orzih/mkdocs-with-pdf
           output_path: '_pdf/PerconaXtradbCluster-8.0.pdf'
           cover_title: 'Percona XtraDB Cluster Documentation'
-          cover_subtitle: 8.0.29-21 (September 12, 2022)
+          cover_subtitle: 8.0.30-22.md (December , 2022)
```

Comment:
I don't think the ".md" should be there. It's on the title page of the PDF.

Reply:
fixed